cookbooks/bcpc-hadoop/templates/default/hdp_hdfs-site.xml.erb

```diff
@@ -15,6 +15,11 @@
   </property>
 
   <property>
+    <name>dfs.datanode.balance.bandwidthPerSec</name>
```

Comment:
Yes, can be converted into a node attribute. I need to figure out a way to get max speed for network card to be able to calculate it based on a ratio.